draft-ietf-quic-transport.md

```diff
@@ -486,9 +486,18 @@ Key Phase Bit:
   recipient of a packet to identify the packet protection keys that are used to
   protect the packet.  See {{QUIC-TLS}} for details.
 
+Google QUIC Demultipexing Bit:
+
+: The fifth bit (0x8) of octet 0 is set to 0. This allows implementations of
+  Google QUIC to distinguish Google QUIC packets from short header packets sent
+  by a client.
+  The special interpretation of this bit SHOULD be removed from this
```

Comment:
This is fine, but you might be better prefixing this with a note to the RFC editor.  You can see one of those in the section on versioning.  In that note, request removal of the bit.  Hopefully we will get to that before this goes to publication.

Reply:
I added the note. Just removing this bit won't be enough though, we'll need to add this bit to whatever greasing mechanism we decide for.

Reply:
In an ideal world, we'd be able to move onto this and then start greasing later after GQUIC has been deprecated.  In reality, I suspect we'll either need a mechanism not to grease certain codepoints in the short header or not to grease at all.  I tend to favor the former, since I think it's generally useful(ie: for WebRTC).